cli/azd/CHANGELOG.md

```diff
@@ -1,15 +1,36 @@
 # Release History
```

Comment:
Can you please:

- Remove 5213
- Remove Breaking Changes section
- Move 5383 to features added

And can you do another round of review and systematically go through each PR listed here and:

1. Gather context from PR description and linked issue
2. Enhance changelog entry and edit CHANGELOG.md immediately. For some bugs, if possible, phrase the changelog entry in terms of the issue that was fixed. E.g. "Fix PowerShell 7 suggestion text not showing for service-level hooks" is a good message.
3. Continue with next PR

Please be exhaustive and process each PR one at a time (do not batch) and **do not** skip any PRs or cut the task short due to time constraints.